vcd/sample_vcd_test_config.json

```diff
@@ -72,5 +72,12 @@
     "//": "Size in megabytes",
     "uploadPieceSize": 5,
     "uploadProgress": true
+  },
+  "envVariables" : {
+    "//" : "Environment variables that we want to set (or unset)",
+    "GOVCD_DEBUG": "",
+    "VCD_SKIP_TEMPLATE_WRITING": "",
+    "VCD_ADD_PROVIDER": ""
```

Comment:
We are at a point where we need to document environment variables in the Terraform project. README.md needs to be updated or even better a separate TESTING.md created with all the nitty details of test setup.

Reply:
Removed testing instructions from `README.md` and added a `TESTING.md` file with explanation of the environment variables.